infrastructure/kubernetes/common/monitoring/README.md

```diff
@@ -18,12 +18,25 @@ following command:
 kubectl create namespace monitoring
 ```
 
+### Setup google Oauth login
+
+##### Set up Google OAuth Application
```

Comment:
I think this subsection is redundant and we can remove it.

Reply:
addressed in: fcf0ade5a8d87a56460dba6164d0810cc5cdc65c